content/automate/ManageIQ/Service/Generic/StateMachines/GenericLifecycle.class/provision.yaml

```diff
@@ -48,3 +48,4 @@ object:
       on_error: update_status(status => 'Error Emailing Owner')
   - finish:
       value: METHOD::finish
+      on_exit: update_status(status => "${/#service_action} Completed Successfully")
```

Comment:
@mkanoor Status here is a message. Will create a followup PR to rename status to message.